project_template/src/index.html

```diff
@@ -18,19 +18,7 @@
         document.querySelector('html').className = 'ios-standalone';
       }
     </script>
-
-<% for (var css of htmlWebpackPlugin.files.css.filter((file) => file.indexOf("shell") != -1)) { %>
-    <style>
-        <%= compilation.assets[css.substr(htmlWebpackPlugin.files.publicPath.length)].source() %>
```

Comment:
With this approach the shell CSS is no longer in the head.
For Cordova apps this doesn't really matter a ton, but for PWAs it's nice to have this.

Not sure if it's worth sticking with this though, as the primary use case for Maji remains Cordova apps. WDYT?